style1.css

```diff
@@ -390,3 +390,285 @@ textarea::placeholder {
 .cont {
   margin: 20px 0;
 }
+
+@media only screen and (min-width: 768px) {
+  #navigation {
+    display: flex;
+    justify-content: space-between;
+    padding: 16px 66px;
+    box-shadow: 0 1px 10px -5px grey;
+    position: fixed;
+    height: 72px;
+  }
+
+  nav {
+    display: flex;
+    justify-content: flex-end;
+  }
```

Comment:
- [ ] Your navigation for the desktop isn't visible right now. Kindly fix it to match the design:
> your design: 
![image](https://user-images.githubusercontent.com/91486791/236466010-4fdf4fac-d587-468f-9251-c680e231aec6.png)
> figma: 
![image](https://user-images.githubusercontent.com/91486791/236466210-603d9e5d-d6f2-4b29-86b6-7f3db56cf690.png)

Reply:
- [ ] Corrected :white_check_mark: